dss/operations/checkout.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+import os
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+
+from dss.storage.identifiers import BUNDLE_PREFIX
+
+logger = logging.getLogger(__name__)
+
+checkout = dispatch.target("checkout", help=__doc__)
+
+
+def verify_delete(handler, bucket, key):
+    logger.warning(f'attempting removal of key: {bucket}/{key}')
+    try:
+        handler.delete(bucket=bucket, key=key)
+        handler.get(bucket=bucket, key=key)
+    except BlobNotFoundError:
+        logger.warning(f'Success! unable to locate key {bucket}/{key} ')
+    else:
+        logger.warning(f'Unable to delete key: {bucket}/{key}')
+        exit(1)
+
+
+@checkout.action("remove_checkout",
+                 arguments={"--replica": dict(choices=[r.name for r in Replica], required=True),
+                            "--keys": dict(nargs="+", help="keys from checkout bucket to remove", required=False),
+                            "--bundle-fqid": dict(nargs="+", help="fqid checkout bucket to remove", required=False)})
```

Comment:
Let's remove `--bundle-fqid`. This operation should conditionally branch on key prefix.